config/newrelic.yml

```diff
@@ -1,22 +1,693 @@
+
+  # Prefix of attributes to include in transaction traces.
+  # Allows * as wildcard at end.
+  # transaction_tracer.attributes.include: []
+
+  # If true, enables collection of transaction traces.
+  # transaction_tracer.enabled: true
+
+  # Threshold (in seconds) above which the agent will collect explain plans.
+  # Relevant only when explain.enabled is true.
+  # transaction_tracer.explain_threshold: 0.5
+
+  # If true, enables the collection of explain plans in transaction traces.
+  # This setting will also apply to explain plans in slow SQL traces if
+  # slow_sql.explain enabled is not set separately.
+  # transaction_tracer.explain.enabled:  true
+
+  # Maximum number of transaction trace nodes to record in a single transaction
+  # trace.
+  # transaction_tracer.limit_segments: 4000
+
+  # If true, the agent records Redis command arguments in transaction traces.
+  # transaction_tracer.record_redis_arguments: false
+
+  # Obfuscation level for SQL queries reported in transaction trace nodes.
+  # By default, this is set to obfuscated, which strips out the numeric and string
+  # literals. If you do not want the agent to capture query information, set this
+  # to 'none'. If you want the agent to capture all query information in its
+  # original form, set this to 'raw'. When you enable high security mode this is
+  # automatically set to 'obfuscated'
+  # transaction_tracer.record_sql: 'obfuscated'
+
+  # Specify a threshold in seconds. The agent includes stack traces in transaction
+  # trace nodes when the stack trace duration exceeds this threshold.
+  # transaction_tracer.stack_trace_threshold: 0.5
+
+  # Specify a threshold in seconds. Transactions with a duration longer than this
+  # threshold are eligible for transaction traces. Specify a float value or the
+  # string apdex_f.
+  transaction_tracer.transaction_threshold: 'apdex_f'
+
+  # If true, the agent automatically detects that it is running in an AWS
+  # environment.
+  # utilization.detect_aws: true
+
+  # If true, the agent automatically detects that it is running in an Azure
+  # environment.
+  # utilization.detect_azure: true
+
+  # If true, the agent automatically detects that it is running in Docker.
+  # utilization.detect_docker: true
+
+  # If true, the agent automatically detects that it is running in an Google Cloud
+  # Platform environment.
+  # utilization.detect_gcp: true
+
+  # If true, the agent automatically detects that it is running in Kubernetes.
+  # utilization.detect_kubernetes: true
+
+  # If true, the agent automatically detects that it is running in a Pivotal Cloud Foundry environment.
+  # utilization.detect_pcf: true
+
+# Environment-specific settings are in this section.
+# RAILS_ENV or RACK_ENV (as appropriate) is used to determine the environment.
+# If your application has other named environments, configure them here.
+development:
+  <<: *default_settings
+  app_name: search.gov (Development)
```

Comment:
We should extract this from the secrets:
```ruby
app_name: <%= Rails.application.secrets.newrelic[:app_name] %> (Development)
```

Reply:
I just checked the cookbooks, and the `app_name` attribute actually already interpolates the environment, so this line can be removed.

Reply:
I added `(Development)` to `secrets.yml.dev` so that we can take out this line entirely.